### PR TITLE
Add [Screen layout] section into Tutorial.md

### PR DESCRIPTION
**Pull request by hyunsik-yoon**
Let's add `[screen layout]` section into `Tutorial.md`.
Additionally, a comment for a candidate `Table of Contents` was added.

ONE-vscode-DCO-1.0-Signed-off-by: Hyun Sik Yoon <hyunsik.yoon.1024@gmail.com>

### CODE REVIEW

File: docs/Tutorial.md [hunk 1]
```diff
@@ -1,6 +1,41 @@
 # Tutorial
 
-Let's compile an example by using ONE-vscode.
+<!-- Candidate order
```
Comment (llFreetimell) [inline docs/Tutorial.md:3]: Is this intentional comments?

File: docs/Tutorial.md [hunk 2]
```diff
@@ -1,6 +1,41 @@
...
+- Opening a folder
+- Installing a toolchain
+- Compiling a model
+- Profiling a model
+
+-->
+
+## Screen layout
+
+ONE-vscode has 7 screen components.
```
Comment (llFreetimell) [inline docs/Tutorial.md:17]:
```suggestion
ONE-vscode has 6 screen components.
```

**Comment by hyunsik-yoon** [comment]
PTAL

**Review by yunjayh — APPROVED** [review]
LGTM

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**